### DISCUSSION TITLE
Security concern

### DISCUSSION THREAD

**Discussion by ChopperDoll** [okibcn/nano-for-windows#26] [General]
No offense anyway, but little bit concern about security of this third-party tools.
Anyone also thinking about this issue?

**Comment by okibcn** [comment]
Security is always a concern, and there are commits related to security. The last one was introduced in release [v7.2-21](https://github.com/okibcn/nano-for-windows/releases/tag/v7.2-21)

Please report any security issues you can find.

**Reply by ghost** [reply]
I have posted in General and "Nano for Windows" has been classified as "Bitdefender GravityZone - Antimalware Event - Trojan.GenericKD.73345737 detected"

**Reply by ghost** [reply]
What other antivirus did detect it, antiviruses are more for appearances you need malware detection then try malwarebytes it's free and trusted more than whatever defender you mentioned? 

Did you check the binary you got from Github?

**Reply by ghost** [reply]
Also even pre https://github.com/okibcn/nano-for-windows/releases/tag/v7.2-21 the arbitrary code execution exploit won't work for those who don't use pyflakes or pylint linters. And the mentioned bug on svannah doesn't include the crafted nanorc for python.

**Comment by rasa** [comment]
The build process is completely transparent, so that should allay worries.

Would it help if the binaries were signed?

**Reply by okibcn** [reply]
If you download them from this repository, then there is no need for that.
